tools/build_chain.sh

```diff
@@ -1551,8 +1555,8 @@ for line in ${ip_array[*]};do
         fi
     fi
     for ((i=0;i<num;++i));do
-        echo "Processing IP:${ip} ID:${i} node's key" >>"${logfile}"
-        local node_count="$(get_value ${ip//./}_count)"
+        echo "Processing IP=${ip} ID=${i} node's key" >>"${logfile}"
+        local node_count="$(get_value ${ip//[\.:]/_}_count)"
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5837713)